suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -147,6 +206,75 @@ const discoverAccountsByDescriptorThunk = createThunk(
     },
 );
 
+export const addNetworkAccountThunk = createThunk(
```

Comment:
@PeKne Was suggesting similar thing, but I think this is just a different form of discovery and the code is strongly tight to the code that is in this file. So to me it felt better to keep it here in case anyone is doing anything with discovery later.